shlr/meson.build

```diff
@@ -2,7 +2,11 @@ res = run_command(py3_exe, '-c', '__import__("sys").exit(__import__("os").path.e
 if res.returncode() == 0
   message('Cloning capstone')
   git_cmd = 'clone -b next --depth 10 https://github.com/aquynh/capstone.git capstone'
-  run_command(git_exe, git_cmd.split())
+  if git_exe.found()
```

Comment:
use `if not git_exe.found()` without `else` branch.

Reply:
Just to clarify if the capstone directory does not exists, I should silently discard this error?

Reply:
If capstone folder doesn't exist and git is not found print error.